src/runtime/String.h

```diff
@@ -288,6 +288,12 @@ class String : public PointerValue {
         return false;
     }
 
+    template <const size_t srcLen>
+    static String* fromASCII(const char (&src)[srcLen])
```

Comment:
Where is it used?

Reply:
This function can used with String::fromASCII("literal")

Reply:
> This function can used with String::fromASCII("literal")

In that case String* fromASCII(const char* s) is used.
This template probably used in situations like below

``` c++
char str[] = "sometext";
String::fromASCII(str);
```

Reply:
Good Point!
I fixed implementation for fixing it